src/app/modules/library/library-project-menu/library-project-menu.component.ts

```diff
@@ -20,13 +21,15 @@ export class LibraryProjectMenuComponent {
   @Input()
   isRun: boolean;
 
+  archived: boolean = false;
```

Comment:
Add access modifier?
```suggestion
  protected archived: boolean = false;
```